commercelayer/dashboard-apps
# Enhance redirect routes behavior

Closes #154

## What I did

Replace app redirects to replace the location.  

SKU and SKU Lists apps. Maybe there's more but those are the ones I found.

This prevents infinite redirects if trying to navigate back in history.

## How to test

Start the server and click SKUs App, navigation should work with appropriate redirect, try to navigate back in history, you should be back to home. Can repeat for SKU Lists App.

## Checklist
